cli/compose/schema/data/config_schema_v3.5.json

```diff
@@ -0,0 +1,544 @@
+{
```

Comment:
Wondering what we did previous times when adding a new version; perhaps it's cleaner if there were two commits; one that adds a "pristine" compose 3.5 (exact copy of 3.4), and one commit that adds your changes